FogCreek/Glitch-Community
# Fallacious slice: First pass at server side rendering

## Links
- https://fallacious-slice.glitch.me/
- https://app.zenhub.com/workspaces/community-5d07d1359dee6059a9688176/issues/fogcreek/glitch-community-work/188

## Changes:
- The homepage and create page both get rendered on the server! Other pages are kept on the client because they would render as just a loading spinner which would (I think) trample the seo stuff
- Introduced babel-register to translate js imported from src/ so it works in node
- Using chokidar to watch the src/ directory and force them to be reimported on next use
(aka hot reloading, because restarting node would also restart the webpack build)
- Several smaller changes around the site as I found stuff that was broken

## How To Test:
- Try refreshing the page around the site. The homepage and create page should be quite a bit faster
(I have it set to production mode so you can do comparisons with community.glitch.me)
- Try loading pages with IE/js disabled/in google translate. They could do some more fancy stuff, but for now I'm keeping it simple and trying to keep the behavior unchanged
